service/src/test/java/bio/terra/profile/service/profile/ProfileServiceUnitTest.java

```diff
@@ -122,7 +123,9 @@ void before() {
             "creator");
     profileDescription =
         new ProfileDescription(
-            profile, Optional.empty(), Optional.of(new Organization().enterprise(false)));
+            profile,
+            Optional.empty(),
+            Optional.of(new Organization().enterprise(false).limits(Map.of())));
```

Comment:
Super nit: `emptyMap` might be _slightly_ clearer.

```suggestion
            Optional.of(new Organization().enterprise(false).limits(emptyMap())));
```

Reply:
Or... could `Organization` default to an empty `limits` map so that this isn't necessary everywhere an `Organization` is created?

Reply:
`Organization` is generated from the swagger code and I haven't found a way to make it default to an empty map instead of null.